src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -302,9 +302,9 @@ namespace AirflowNetwork {
         void calculate_GC_balance();
         void mrxinv(int NORDER);
         void report();
-        void update(Optional_bool_const FirstHVACIteration = _); // True when solution technique on first iteration
-        void venting_control(int i,                              // AirflowNetwork surface number
-                             Real64 &OpenFactor                  // Window or door opening factor (used to calculate airflow)
+        void update(ObjexxFCL::Optional_bool_const FirstHVACIteration = _); // True when solution technique on first iteration
+        void venting_control(int i,                                         // AirflowNetwork surface number
+                             Real64 &OpenFactor                             // Window or door opening factor (used to calculate airflow)
```

Comment:
Interesting that Clang-Format felt inclined to indent the comments on a separate function declaration, but :shrug: